app/form_objects/support_ticket/contact_details_form.rb

```diff
@@ -5,4 +5,6 @@ class SupportTicket::ContactDetailsForm
 
   validates :full_name, presence: { message: 'Enter your full name' }
   validates :email_address, presence: { message: 'Enter your email address' }
+  validates :email_address, format: { with: URI::MailTo::EMAIL_REGEXP, message: "You must enter an email address in the correct format, like name@example.com"}
```

Comment:
```suggestion
  validates :email_address, email_address: true
```
which would use the existing `EmailAddressValidator`.